From 12b4010768d5bf36bdc1100204be1a9de913d4f4 Mon Sep 17 00:00:00 2001
From: Vaibhav Hiremath <hvaibhav@ti.com>
Date: Thu, 15 Jul 2010 13:53:03 +0530
Subject: [PATCH 44/75] mt9v113: AE param tuned to get 28~30FPS

Please note that with this change we are compromising with
image quality, so should be used only in case of high FPS use-case.
---
 drivers/media/video/mt9v113.c |    8 ++++++++
 1 files changed, 8 insertions(+), 0 deletions(-)

diff --git a/drivers/media/video/mt9v113.c b/drivers/media/video/mt9v113.c
index 466702e..6714240 100644
--- a/drivers/media/video/mt9v113.c
+++ b/drivers/media/video/mt9v113.c
@@ -306,6 +306,14 @@ static struct mt9v113_reg mt9v113_reg_list[] = {
 	{TOK_WRITE, 0x0990, 0x0003},
 	{TOK_WRITE, 0x098C, 0xA410},
 	{TOK_WRITE, 0x0990, 0x000A},
+
+	{TOK_WRITE, 0x098C, 0xA20C},
+	{TOK_WRITE, 0x0990, 0x0003},
+	{TOK_WRITE, 0x098C, 0xA20B},
+	{TOK_WRITE, 0x0990, 0x0000},
+	{TOK_WRITE, 0x098C, 0xA215},
+	{TOK_WRITE, 0x0990, 0x0004},
+
 	{TOK_WRITE, 0x098C, 0xA103},
 	{TOK_WRITE, 0x0990, 0x0006},
 	{TOK_DELAY, 0, 100},
-- 
1.6.6.1

